.quiet_quality.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
I've been enjoying `logging: light` in qq - how would you feel about that? I pretty much always need to run the actual failing tool after I see failures right now, since the real tool output usually has better clues (and failures are normally from rspec - standard tells me in my editor and I rarely write methods that cross the complexity lines)

Reply:
yeah that sounds like a good idea!